libs/auth-api-lib/src/lib/delegations/delegations-incoming.service.ts

```diff
@@ -182,6 +188,17 @@ export class DelegationsIncomingService {
       )
     }
 
+    // This has a provider of Custom but needs to be handled separately
```

Comment:
Custom is also provider of GeneralMandate, we should explicitly check the custom delegation type in the if above.